### PR TITLE
Added json file for yash-0025

### PR DESCRIPTION
**Pull request by yash-0025**
Added Json file for yash-0025

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1065"></a>

### CODE REVIEW

File: public/data/yash-0025.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/yash-patel-9343601b7/",
+        "icon": "linkedin"
+      }
+    ]
+  }
+
```
Comment (syedsohan) [inline public/data/yash-0025.json:24]:
```suggestion
```

File: public/data/yash-0025.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        "url": "https://twitter.com/YASH80914470",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/yash-patel-9343601b7/",
+        "icon": "linkedin"
+      }
+    ]
+  }
```
Comment (syedsohan) [inline public/data/yash-0025.json:23]:
```suggestion
}
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Review by KendallDoesCoding — COMMENTED** [review]
Maybe consider the file name being YashPatil.json or Yash0025.json

According to me both these look good, instead of the - and capitalize text properly so pick one you like.

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, thanks for your contribution :+1:

**Review by syedsohan — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**